### ISSUE TITLE
Cannot use nutanix_subnets data source

### ISSUE THREAD

**Issue by franc01s** [nutanix/terraform-provider-nutanix#73]
**Describe the bug**
Cannot use nutanix_subnets datasource using a simple configuration
```
provider "nutanix" {
  username = "admin"
  password = "password"
  endpoint = "its-prism-central-lab"
  insecure = true
  port = 9440
  wait_timeout = 10
  //Optional
}
data "nutanix_clusters" "clusters" {
}
data "nutanix_subnets" "subnets" {
}
```

**Expected behavior**
terraform plan running without errors

**Logs**
```
➜  terraform terraform plan

Error: Invalid data source

  on main.tf line 43, in data "nutanix_subnets" "subnets":
  43: data "nutanix_subnets" "subnets" {

The provider provider.nutanix does not support data source "nutanix_subnets".
```

**Versions (please complete the following information):**
 - OS 
```
OSX ➜  10.14.5
```
 - Terraform
```
Terraform v0.12.4
+ provider.nutanix (unversioned)
```
 - Nutanix Cluster
``` 
5.10.5
```
 - Nutanix Prism Central
```
5.10.5
```
 - Terraform provider version (using posted RC1? Compiling yourself? If so, from Develop or Master, and what commit id?
```
commit b22b01e704a7009f45676a0af662f58a31a78832 (HEAD -> tf-0.12-suppport, origin/tf-0.12-suppport)
Author: Marin Salinas <marinssalinas@gmail.com>
Date:   Wed Jul 17 12:22:01 2019 -0500
```

**Comment by coderGo93** [comment]
Hello @franc01s, I'd like to know if this issue is still happening on branch master(newest version)? just to be sure, please

**Comment by franc01s** [comment]
Hello @coderGo93 ,
Yes still the same.
Terraform v0.12.24
+ provider.nutanix v1.0.2

**Comment by coderGo93** [comment]
Hello @franc01s, thanks for you reply. 
Yesterday I made [PR](https://github.com/terraform-providers/terraform-provider-nutanix/pull/115) with branch `fix-#73`, could you please try it again if it works for you on branch `fix-#73`?

**Comment by franc01s** [comment]
I can confirm that works now.
Thanks @coderGo93
